django_socio_grpc/management/commands/generateproto.py

```diff
@@ -11,6 +11,13 @@
 from django_socio_grpc.protobuf.generators import RegistryToProtoGenerator
 from django_socio_grpc.settings import grpc_settings
 
+try:
+    from mypy_protobuf.main import __version__  # noqa: F401
+
+    mypy_protobuf_installed = True
```

Comment:
@legau 
What do you think on this. Is it better to have a global variable here with the import at the top of the file or trying the import only if the --with-types options is True but putting the import inside the code ?